ginivision/src/doc/source/changelog.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+2.1.0 (2017-01-30)
+==================
+
+- Removed the 4:3 aspect ratio camera resolution requirement. Only a minimum of 8MP is required.
+- Removed the continuous-focus mode requirement. Only auto-focus is required.
+- If no continuous-focus mode is available the camera is requested to do an auto-focus run before taking a picture.
```

Comment:
+- If no continuous-focus mode is available then an auto-focus run is triggered when the user activates the capture button.